simp_le.py

```diff
@@ -371,7 +396,8 @@ def load(self):  # pylint: disable=missing-docstring
     def save(self, data):  # pylint: disable=missing-docstring
         logger.info('Saving %s', self.path)
         output = (self.dump_cert(chain_cert) for chain_cert in data.chain)
-        with open(self.path, 'wb') as chain_file:
+        open_func = open_keyfile if data.key else open
+        with open_func(self.path, 'wb') as chain_file:
```

Comment:
Ok, apparently a ChainFile behaves likes its name, it does not store the key even though the FullChainFile implementation suggests otherwise.

This becomes different with a change such as #33, should it always set the permission to 0600 for security reasons? If the user requires weaker permissions, the file can be touched before creating (or chmod can be invoked afterwards).